### ISSUE TITLE
audio streams done with VideoStream API don't join-in-progress

### ISSUE THREAD

**Issue by sorenhan** [microsoft/mixed-reality-extension-sdk#509]
From Steven: when using a video stream instance that plays an audio stream (for example a radio feed), users joining the MRE late don't hear the stream

One guess is because join-in-progress on regular video streams seek forward, the seek may be not working on live streams.

**Comment by norybiak** [comment]
It turns out that any live media (online radio, Mixer) will have a defined duration of `0`.

The `set-media-state` rule is responsible for setting the the [expirationTime of the media instance](https://github.com/microsoft/mixed-reality-extension-sdk/blob/409bdaf08e7ac0883e20db2ce281670d78245cb9/packages/sdk/src/internal/adapters/multipeer/rules.ts#L1089). Because it only checks to see if the VideoStream duration is `undefined`, a duration of `0` means the instance will receive an expirationTime of the current time (basisTime).

Within the `activeMediaInstances` method in clientSync.ts, the following [line of code](https://github.com/microsoft/mixed-reality-extension-sdk/blob/409bdaf08e7ac0883e20db2ce281670d78245cb9/packages/sdk/src/internal/adapters/multipeer/protocols/clientSync.ts#L301) checks to see if the media instance has expired. It will always be true because the expirationTime is defined and it will always be slightly less than the targetTime.
```javascript
if (activeMediaInstance.expirationTime !== undefined && targetTime > activeMediaInstance.expirationTime)
``` 
The sync payload never gets sent.

Possible fix is to check if the duration is `0` in the `set-media-state` rule.
```javascript
if (activeMediaInstance.message.payload.options.looping === true ||
    activeMediaInstance.message.payload.options.paused === true ||
    (asset === undefined || asset.duration === undefined || asset.duration === 0)) {
        // media instance current will last forever
        activeMediaInstance.expirationTime = undefined;
}
else {
```

**Comment by stevenvergenz** [comment]
Fixed by #569
